src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -769,16 +773,14 @@ private async Task AppendAsyncInternal(DateTime timestamp, IEnumerable<double> v
                             _operation._currentWriter.Write(value);
```

Comment:
This value is a `double` which in German culture is `3,14` instead of `3.14`.
I _think_ it might be a problem and a test for it would be nice :-)

Reply:
done

Reply:
We *must* write the values using culture insensitive manner